test/karma.config.ts

```diff
@@ -57,13 +58,24 @@ export function config(config) {
     customLaunchers: customLaunchers,
 
     exclude: [],
-    preprocessors: {},
-    reporters: ['dots'],
+    preprocessors: {
+      'dist/**/!(*spec).js': ['coverage']
+    }
+    reporters: ['dots', 'coverage'],
```

Comment:
It was something I was going to work out with you before I PR the typescript coverage, one of the two will have to change.